store/rootmulti/store.go

```diff
@@ -153,6 +160,11 @@ func (rs *Store) StoreKeysByName() map[string]types.StoreKey {
 	return rs.keysByName
 }
 
+// GetCommitKVStores get all kv stores associated wit the multistore.
+func (rs *Store) GetCommitKVStores() map[types.StoreKey]types.CommitKVStore {
+	return rs.stores
```

Comment:
This value is read-from and written-to without any kind of lock. Is that safe?

Reply:
This method is unused, I removed it